extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduOperations.java

```diff
@@ -8,6 +8,9 @@
  */
 package org.locationtech.geowave.datastore.kudu.operations;
 
+import com.google.common.collect.Iterators;
```

Comment:
Why are you importing this from `google` instead of the java standard libraries?

Reply:
The java standard lib doesn't have iterator concat